sklearn/utils/tests/test_validation.py

```diff
@@ -853,3 +855,28 @@ def test_check_scalar_invalid(x, target_name, target_type, min_val, max_val,
                      min_val=min_val, max_val=max_val)
     assert str(raised_error.value) == str(err_msg)
     assert type(raised_error.value) == type(err_msg)
+
+
+def test_check_sample_weight():
```

Comment:
I think that we need additional tests to ensure that we pass correctly the `kwargs` to `check_array`

Reply:
Actually kwargs are not longer needed, so I remove them. They can always be added later if necessary.